src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/20210323093442_AddIndexes.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Migrations
+{
+    public partial class AddIndexes : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateIndex(
```

Comment:
I think this should be verified with @gonesvik  as well. We have no clustered indexes. Should we analyse a bit more i detail which (or if) clustered indexes should be added?

Reply:
Why do this become a clustered index @paal123 ?
It was not ment to be. Trying to set up indexed as in presevation (since speed is currently ok there), created by Demissie. But in presevation D added all columns in index which was back then mentioned as incorrect ... so what is the correct to do here, to set up non-clustered indexes with correct columns added?